Dockerfile

```diff
@@ -23,10 +20,6 @@ RUN apt-get update && \
         android-tools-adb \
         git \
         python-pip \
-        build-essential \
-        cmake \
-        ca-certificates \
-        libc6:i386 \
```

Comment:
Please leave these packages in a comment. Just not to search for them in future.

Reply:
Done.